AMM/contracts/AMM/tests/functions/add_pool.rs

```diff
@@ -0,0 +1,119 @@
+use crate::utils::{
+    amm_abi_calls::{add_pool, pool},
+    test_helpers::{deploy_and_construct_exchange_contract, setup, setup_and_initialize},
+};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn adds() {
```

Comment:
Is this test implying that the pair is ordered?

Reply:
This test and the other test combined imply that the order of the pair doesn't matter while adding.